aspnetcore/mvc/models/validation/samples/ValidationResultErrorMessage/ContactsController.cs

```diff
@@ -151,14 +146,18 @@ public async Task<IActionResult> DeleteConfirmed(Guid id)
             {
                 _context.Contact.Remove(contact);
             }
-            
+            else
+            {
+                return Problem("Contact deleted.");
+            }
+
```

Comment:
I added this to make it easy to test. Click open in new tab on delete button. Then delete the item. In new tab, try delete again.

Reply:
ok let me see

Reply:
I could return the Error View while serializing the JSON into the Error model.
![image](https://user-images.githubusercontent.com/36219292/236105890-0a83fcb6-8fe3-4dfc-8f0a-10d30410147f.png)
@Rick-Anderson